src/Context/authContext.js

```diff
@@ -0,0 +1,128 @@
+import { createContext, useContext, useReducer } from "react";
+import axios from "axios";
+
+export const InitialAuthData = {
+  authData: {},
+  authErrorMsg: "",
+};
+
+export const AuthReducer = (state, { type, payload }) => {
```

Comment:
you can have a different file for reducers combined

Reply:
Thankyou. Will surely implementing it.